### DISCUSSION TITLE
Does this look right? Page content seems very narrow.

### DISCUSSION THREAD

**Discussion by Tyree** [searxng/searxng#4100] [Q&A] [answered]
Shouldn't the interface use more of the browser width than this?
Is there a setting somewhere for this? I've been trying to google it, but I'm not coming up with anything.
Thanks!
![Screenshot 2024-12-10 091435](https://github.com/user-attachments/assets/c5e59a1f-33d2-445f-93e4-571ce18476d5)

**Comment by Aadniz** [comment, marked as answer]
This is nothing you can change in the settings. Maybe the closest thing you are looking for would be `Center Alignment` found in Preferences -> User Interface. Looks like this:

![image](https://github.com/user-attachments/assets/7c4f87b4-45ba-4eb9-a3fe-a96578f5810a)

Even if it was possible to stretch it out fully, would it even look pleasing?

![image](https://github.com/user-attachments/assets/dbfc0720-0d2d-4680-96ec-1be897ed0a72)

I agree that current style of search engines has a big margin of nothing on it, and it is surely something I would love to see tuned too.

*(Not knowing the context if you are hosting your own instance or using someone else's)* For now, the best solution you can do is to install an extension that alters the CSS of the webpage, giving you a more stretched result, or you could try find an instance that has their own custom CSS on it, or you could run your own fork with altered CSS and lastly you could contribute to the project with your own CSS solution.

Also obviously, zooming in, having it at 110% or 125% for example.

**Reply by Tyree** [reply]
I am self-hosting. And yeah, I agree, stretching out all the way like you've shown is too much. But dang, it feels like it's using max-width from 1996 when the most prevalent screen resolutions were 800x600 or 1024x768. :-D 
That said, Google isn't much better (see below). 
Maybe I can figure a way to hack the CSS and give it a little breathing room. 100px or so. :-) (Anyone figured out a way to do that?)
Thanks!
![Screenshot 2024-12-11 064726](https://github.com/user-attachments/assets/4a1a142b-d3b8-4824-a215-3cc9b2dd5007)

**Reply by Tyree** [reply]
Found an extension (Chrome and Brave) called CSS Inject that saves CSS overrides per url. Much better (IMO):
![Screenshot 2024-12-11 070155](https://github.com/user-attachments/assets/9db070a5-705b-427e-bccc-9cd6840b9b91)

**Reply by Aadniz** [reply]
Changing this style for your instance, you can alter this `@results-width` less variable, then recompile static https://github.com/searxng/searxng/blob/master/searx/static/themes/simple/src/less/definitions.less#L272 Try change it to `75rem` for example.

**Reply by Tyree** [reply]
Admittedly, I have not messed with SearXNG much at all (just looking into the docs a bit this morning - complicated). I have it installed on unRAID via Community Apps plugin. I am not knowledgeable enough to tinker with it. The settings file alone that comes with the docker container via Community Apps is LONG. 
I'm also getting a LOT of errors in the logs about problems loading engine py files. I commented out some of the engine blocks in the settings, but still lots of errors in the container log.
Search works, but for instance, autocomplete doesn't work even though it's set to duckduckgo in the settings.
So yeah, my instance has worse problems than the width of the results. :-D 
And the documentation is overwhelming to a novice like myself. And compiling or recompiling makes my head hurt. :-D
Thanks!

**Reply by ingenarel** [reply]
> Even if it was possible to stretch it out fully, would it even look pleasing?

@Aadniz sorry for necrobumping this do you by any chance know how to make that work in the latest version?

my instance is self hosted using docker and i've been hacking with the theme a little bit by just replacing the simple theme with a fork of it (altho my css knowledge is extremely limited)


i have checked for a results-width var altho there isn't one in the latest sources, was wondering how to do that exact same thing

**Reply by Aadniz** [reply]
@ingenarel Change the `@results-width: 45rem` to `@results-width: 75rem` for example in [definitions.less#L282](https://github.com/searxng/searxng/blob/34e993a85431c8a84c5ce7071281c7e198fceade/client/simple/src/less/definitions.less#L282), then run `make themes` (It might require NodeJS and a few other JavaScript tools to work), and then restart SearXNG. You might also want to clear the cache in your browser when testing, just do CTRL + F5 to refresh without caching the newly compiled CSS.

**Reply by ingenarel** [reply]
ahh i was actually messing around with the output files in `searx/static/themes/simple` instead of the src, will try to test them out, thanks!
